modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/AmqpConnectionManager.java

```diff
@@ -69,4 +70,21 @@ public void removeConnectionHandler(AmqpConnectionHandler handler) {
             return new ArrayList<>(connectionHandlers);
         }
     }
+
+    /**
+     * Closes an AMQP connection specified by the identifier.
+     *
+     * @param id the connection identifier
+     */
+    public void closeConnection(int id) {
```

Comment:
What if we change the name to `forceDisconnect`? since that's what we do under the hood.